data/README.md

```diff
@@ -0,0 +1,100 @@
+# Data
+
+A guide for managing a series of tubes.
+
+## Best Practices
+
+* Use stream processing for data which is not guaranteed to fit into memory.
+
+### Streaming
+
+* Avoid automatic retries for non-idempotent operations.
+* Avoid storing messages with different schemas in the same topic or message
```

Comment:
"Topic" is referenced a couple of times here - would it be worth defining it in the glossary?

Reply:
I added some glossary entries for these in 7cf2746.